company/index.html

```diff
@@ -0,0 +1,185 @@
+                  <span aria-hidden="true">x</span>
+                  <span class="sr-only">Close</span>
+                </button>
+                <h3>{{member.name}}<br> <small>{{member.role}}</small></h3>
+              </div>
+              <div class="modal-body">
+                <p><img class="img-responsive margin-bottom-30 profile-image" src="{{member.image}}" alt="{{member.name}}">
+                </p>
+                {{member.fullDescription}}
+                <ul class="list-unstyled list-inline team-socail">
+                  <li><a href="http://twitter.com/{{member.twitter}}"><i class="fa fa-twitter fa-3x"></i></a></li>
+                  {% if member.github != null %}
+                  <li><a href="http://github.com/{{member.github}}"><i class="fa fa-github fa-3x"></i></a></li>
+                  {% endif %}
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+        {% endfor %}
+      </div>
+
+      <div class="row margin-bottom-40">
+        <div class="headline"><h2>Our culture</h2></div>
+        <p>Our craftsmen, craftswomen, and apprentices are at the core of our company. We call ourselves craftsmen and craftswomen as a way to express the passion we have for creating well-crafted software. As most of us came to Codurance through the Craftsmanship communities (London and a few others in Europe), it is in our DNA to run many internal sessions where we learn from and share knowledge with each other. Our people are never alone, even when they are working with a client. We are constantly helping each other, regardless which project we are working on.</p>
+
+        <p>Transparency is one of our key values. Every person inside Codurance has full access to the company's financials and each other salaries. There are no hierarchies or managers in the company. Any small group of people who feel passionate about a subject or want to change how things are done in the company can start an Initiative. The group consider all the people who may be impacted by their decision and accept any person interested in the subject to join the group. The group is then empowered to make the decision.</p>
+
+        <p>Agile and Lean principles are also at our core&mdash;any decision made now can be changed later. This allows us to always move forward, try new things, inspect and adapt.</p>
+
+      </div>
+     
+      <div class="row margin-bottom-40">
+        <div class="headline"><h2>How we work</h2></div>
+        <p>The way we work is firmly grounded in Software Craftsmanship and Agile principles. We believe that software should continuously add value to the business with inherent quality that allows it to evolve along with the changing needs of the business.</p>
+
+
+        <div class="row news-v1">
+          <div class="col-md-4 md-margin-bottom-40">
+            <div class="news-v1-in bg-color-white">
+              <h3 class="font-normal"><a href="#">Software Craftsmanship</a></h3>
```

Comment:
All the headers in the "How we work" sections link to an anchor on the top of the page. I'd say that the headers are better without links, as there is no content to link them to.